test/unit/views/StatementPageSpec.scala

```diff
@@ -151,11 +151,13 @@ class StatementPageSpec extends AnyWordSpec with Matchers with GuiceOneAppPerSui
     }
 
     "return HTML that contains the correct `What to do if you have difficulty using this service` section" in new FullSetup {
-      fullyAccessibleCHGVStatementHtml should include
-      """<h2 class="govuk-heading-l">What to do if you have difficulty using this service</h2>"""
```

Comment:
without the brackets, this was not actually asserting anything - noticed a compiler warning for this